packages/amplify-graphql-api-construct/src/types.ts

```diff
@@ -139,6 +179,12 @@ export interface AuthorizationModes {
    */
```

Comment:
L177 above now has an incomplete comment. We should work with @renebrandel for wording, but maybe something like "IAM Auth config, required if an 'iam' auth provider is specified in the API, or if specifying that all IAM authorization should be controlled by IAM policies rather than `@auth` directives."

Reply:
updated.
I didn't elaborate about `iam` provider since we're deprecating it and moving that config to `IdentityPoolAuthorizationConfig`

Reply:
it's not a `rather` as the use cases are slightly different. I'd suggest this:
```suggestion
IAM Auth config, required to allow IAM-based access to this API. This applies to any IAM principal except Amazon Cognito identity pool's authenticated and unauthenticated roles. This behavior was has recently been improved. See https://docs.amplify.aws/cli/react/tools/cli/migration/iam-auth-updates-for-cdk-construct for details.
```